PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -18,25 +18,26 @@ def makePuppies( process ):
 
 
     process.puppiNoLep = process.puppi.clone()
-    process.puppiNoLep.candName = cms.InputTag('pfNoLepPUPPI') 
-
-    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
+    process.puppiNoLep.candName = cms.InputTag('particleFlow')
+    process.puppiNoLep.useWeightsNoLep = cms.bool(True)
+    
+#    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
```

Comment:
@slava77 the modification in the puppiForMET_cff should come from the PuppiPR #15429 #15562 and not from this PR.
Would be good to merge Puppi's PR first and remove the modification from here